02-starting-with-data.Rmd

```diff
@@ -197,40 +323,55 @@ Notice that in the `levels()` approach, three important steps occur:
 * We then access these numeric values using the underlying integers of the
   vector `f` inside the square brackets
 
-### Challenge
+### Renaming factors
 
-The function `plot()` can be used to quickly create a bar plot of a factor. For instance, for the factor
-`exprmt <- factor(c("treat1", "treat2", "treat1", "treat3", "treat1", "control", "treat1", "treat2", "treat3"))`, the code `plot(exprmt)`
-gives you a barplot of the number of observations at each level, as shown
-below.
+When your data is stored as a factor, you can use the `plot()` function to get a
+quick glance at the number of observations represented by each factor
+level. Let's look at the number of males and females captured over the course of
+the experiment:
 
-* What determines the order in which the treatments are listed in the plot? (Hint: use `str` to inspect the factor.)
-* How can you recreate this plot with "control" listed last instead
-of first?
+```{r, purl=TRUE}
+## bar plot of the number of females and males captured during the experiment:
+plot(surveys$sex)
+```
 
+In addition of males and females, there are about 1700 individuals for which the
+sex information hasn't been recorded. Additionally, for the these individuals,
+there is no label to indicate that the information is missing. Let's rename this
+label to something more meaningful. Before doing that, we are going to create a
+new object so we don't loose the original data:
```

Comment:
suggest something like, instead of Before doing...

'We're going to pull out the data on sex and work with that data, so we're not modifying the working copy of the data frame'